client/luigi-client.js

```diff
@@ -39,205 +37,220 @@
     });
   }
 
-  var Luigi = {};
-  var currentContext = {};
+  // actual Luigi code
+  var luigiInitialized = false
   var defaultContextKeys = ['context', 'internal', 'nodeParams', 'pathParams'];
-  var _contextUpdated;
+  var currentContext = defaultContextKeys.reduce(function(acc, key) {
```

Comment:
`Array.reduce` is not available with ES5

Reply:
Actually it is :) https://developer.mozilla.org/en-US/docs/Web/JavaScript/Reference/Global_Objects/Array/reduce#Specifications

Reply:
OMG, never seen it before ES6. Thanks for the info!